pkg/util/cmdutil/console.go

```diff
@@ -95,6 +95,16 @@ func RemoveTrailingNewline(s string) string {
 	return s
 }
 
+// EndKeypadTransmitMode switches the terminal out of the keypad transmit 'application' mode back to 'normal' mode.
+func EndKeypadTransmitMode() {
+	if runtime.GOOS != "windows" && Interactive() {
```

Comment:
On the “Windows” part - is that the right test, or is actually something about whether a certain kind of terminal is attached?  Are there Windows environments where this same issue will still occur?

Reply:
So I double-checked and `GOOS` returns `linux` on WSL. Cmd/PowerShell terminals have no notion of normal/application mode and tput commands, AFAIK.